src/viur/shop/payment_providers/abstract.py

```diff
@@ -28,6 +28,16 @@ def title(self) -> translate:
         """Define the external title of the payment provider"""
         return translate(f"viur.shop.payment_provider.{self.name}", self.name)
 
+    @property
+    def description(self) -> translate:
+        """Define the description of the payment provider"""
+        return translate(f"viur.shop.payment_provider.{self.name}.descr", self.name)
+
+    @property
+    def image_path(self) -> str | None:
```

Comment:
I would solve this via the init, otherwise you cannot set this in a nice way:
```py
    def __init__(
        self,
        *,
        image_path: str | None = None,
    ):
        super().__init__()
        self.image_path = image_path
```